ptz-explainer.md

```diff
@@ -0,0 +1,166 @@
+user will be prompted. Otherwise, the pan and tilt values passed in
+`applyConstraints()` will be used to move the camera.
+
+The example below shows how camera pan and tilt could be requested and presented
+to the user.
+
+```js
+// User is prompted to grant camera PTZ access only if the camera supports PTZ.
+// If the camera does not support PTZ, it falls back to a "regular" camera prompt.
+const videoStream = await navigator.mediaDevices.getUserMedia({
+  video: {
+    // [NEW] Website asks to control camera PTZ.
+    pan: true, tilt: true, zoom: true, 
+  }
+});
+
+// Show camera video stream to user.
+const video = document.querySelector("video");
+video.srcObject = videoStream;
+
+// Get video track capabilities and settings.
+const videoTrack = videoStream.getVideoTracks()[0];
+const capabilities = videoTrack.getCapabilities();
+const settings = videoTrack.getSettings();
+
+// [NEW] If the camera supports pan motion, let the user control it.
+if ("pan" in capabilities) {
+  const input = document.querySelector("input[type="range"]");
+  input.min = capabilities.pan.min;
+  input.max = capabilities.pan.max;
+  input.step = capabilities.pan.step;
+  input.value = settings.pan;
+  input.oninput = async (event) => {
+    // [NEW] Applying PTZ constraints requests the permission
+    // descriptor `{name: "camera", panTiltZoom: true}`. Since this is already
+    // granted during successful getUserMedia call above, this applyConstraints
+    // call should not prompt for any new permissions.
+    await videoTrack.applyConstraints({
+      advanced: [{ pan: event.target.value }],
+    });
+  };
+}
+
+// [NEW] If the camera supports tilt motion, let the user control it.
+if ("tilt" in capabilities) {
+  // similar to pan motion above.
+}
+```
+
+[Spec PR](https://github.com/w3c/mediacapture-image/pull/218)
+
+## Integration with the Permissions API
+
+Having a separate PTZ permission allows the UA to differentiate between normal
+camera permissions and PTZ camera permissions as PTZ needs to be explicitly
+requested as an extension to the camera permission.
+
+UA may group both the camera permission request and the PTZ permission request.
+See mock below.
+
+![Mock of a PTZ permission prompt](/images/ptz-prompt-mock.png)
+
+The camera permission can be requested using `{name: "camera", panTiltZoom:
+false}` or `{name: "camera"}` while the PTZ permission can be requested using
+`{name: "camera", panTiltZoom: true}`. The latter is used in `getUserMedia()` if
+`pan`, `tilt`, and `zoom` values are "true".
+
+The example below shows how to query the current status of the PTZ permission.
+
+```js
+const panTiltZoomPermissionStatus = await navigator.permissions.query({
+  name: "camera",
+  panTiltZoom: true,
+});
+
+if (panTiltZoomPermissionStatus.state == "granted") {
+  // User has granted access to this website to control camera PTZ.
+}
+```
+
+To be explicit, `{name: "camera", panTiltZoom: true}` is stronger than `{name:
```

Comment:
I don't see exactly how you would be able to group them like in my examples above. If that is posssible, it would be good to spell it out. It seems right now that I will get one failed prompt and then have to reprompt, which will just result in the prompt-fatigue argument